pkg/client/rpc_client.go

```diff
@@ -570,9 +573,17 @@ func (r *RPCClient) HeaderByHash(ctx context.Context, hash common.Hash) (header
 func (r *RPCClient) LatestSafeBlock(ctx context.Context) (head *evmtypes.Head, err error) {
 	ctx, cancel, _, _, _ := r.acquireQueryCtx(ctx, r.rpcTimeout)
 	defer cancel()
-	err = r.ethGetBlockByNumber(ctx, rpc.SafeBlockNumber.String(), &head)
-	if err != nil {
-		return
+
+	if r.finalityTagEnabled && r.safeTagEnabled {
```

Comment:
IMO, this log should be done [here](https://github.com/smartcontractkit/chainlink-framework/blob/2350c82883e20adf2d0ae5ce482a90993b3e6b74/chains/heads/tracker.go#L404).
RPC should always return what was requested, even if chain does not support that.

Reply:
done